Description: fix unicode version
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-09-16

--- a/build.js
+++ b/build.js
@@ -4,7 +4,7 @@
 const jsesc = require('jsesc');
 const emptyDirSync = require('fs-extra').emptyDirSync;
 const regenerate = require('regenerate');
-const UNICODE_VERSION = require('./unicode-version.js');
+const UNICODE_VERSION = '15.0.0';
 const unicode = require(`@unicode/unicode-${ UNICODE_VERSION }`);
 
 /*----------------------------------------------------------------------------*/
